### ISSUE TITLE
Different color/shape for inheritance/inclusion/extension dependencies

### ISSUE THREAD

**Issue by dcadenas** [dcadenas/rubydeps#3]
Show in a different color and/or different shape those dependencies that are from a child class to it's parent class.

**Comment by aspiers** [comment]
Yes, that would be awesome.  Any chance of this any time soon?  If not, if you give me one or two pointers I might be able to do this myself.
